otri/database/database_stream.py

```diff
@@ -0,0 +1,80 @@
+import psycopg2
+
+from .database_query import DatabaseQuery
+
+
+class DatabaseStream():
+
+    def __iter__(self):
```

Comment:
Is this class an `Iterator` or an `Iterable`? You shouldn't mix the two things by using both `__iter__` and `__next__`